composer.json

```diff
@@ -2,18 +2,18 @@
   "name": "magento/composer",
   "description": "Magento composer library helps to instantiate Composer application and run composer commands.",
   "type": "library",
-  "version": "1.2.0",
+  "version": "1.3.0",
   "license": [
     "OSL-3.0",
     "AFL-3.0"
   ],
   "require": {
-    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0",
-    "composer/composer": "1.4.1",
-    "symfony/console": "~2.3, !=2.7.0"
+    "php": "~5.5.0|~5.6.0|~7.0.0|~7.1.0|~7.2.0",
+    "composer/composer": "~1.6.0",
+    "symfony/console": "~4.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "4.1.0"
+    "phpunit/phpunit": "~6.2.0"
```

Comment:
As PHPUnit 7 has been released, can we update to `~7.0.0`? Looks like it doesn't cause dependency conflicts and tests pass.